shared/pingpong/pingpong.go

```diff
@@ -234,6 +236,31 @@ func fundAccounts(accounts map[string]uint64, client libgoal.Client, cfg PpConfi
 			throttleTransactionRate(startTime, cfg, totalSent)
 		}
 	}
+	// wait until all the above transactions are sent, or that we have no more transactions
+	// in our pending transaction pool.
+	for {
```

Comment:
Does this assume that all nodes have the same set of transactions in their transaction pool?

Reply:
No - on this branch, nodes that do no have any online stake do not receive any transactions from the network. But I get your point - I'll revise the code and send it for review later on.